timely_beliefs/tests/test_belief_io.py

```diff
@@ -69,6 +84,25 @@ def test_load_beliefs(csv_file):
     assert isinstance(bdf.index.get_level_values("belief_time")[0], datetime)
 
 
+def test_load_beliefs_with_nat_values(csv_nat_file):
+    """Test loading BeliefsDataFrame from csv containing one NaT belief_time value.
+
+    The NaT value is an empty cell in the belief time column.
+    The relevant row should be skipped, as requested using the combination of na_values and keep_default_na.
+    """
+
+    # Load beliefs with tb.read_csv
+    df = tb.read_csv(
+        "test_NaT.csv",
+        sensor=tb.Sensor("Sensor Y"),
+        source=tb.BeliefSource("Source A"),
+        timezone="Europe/Amsterdam",
+        na_values=[""],
```

Comment:
Do we need this line here? I believe there is no na values as we skip them.

Reply:
We do. This tells `pd.read_csv` to treat empty cells as NaN values.